index.html

```diff
@@ -4,9 +4,63 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" type="text/css" href="css/style.css">
-  <title>Hello Front-End</title>
+  <title>Airbnb</title>
 </head>
 <body>
-  <h1>Hello Front-End</h1>
+
+  <main>
+    <h1>Meet Guidebooks</h1>
+    <h2>Discover hundreds of local spots recommended by Airbnb hosts</h2>
+
+<div class="flex-container">
+  <div class="img">
+      <img src="img/san-francisco.jpg">
+      <div class="txt">
+        <p>San Francisco</p>
+      </div>
+  </div>
+  <div class="img">
+      <img src="img/new-york.jpg">
```

Comment:
An alternate approach to using an `<img>` tag here would be to set that image as the `background-image` of the `<div>`. Then the only HTML content of the `<div>` would be the `<p>`, which you can center horizontally and vertically using flexbox.